sdk/devcenter/azure-developer-devcenter/CHANGELOG.md

```diff
@@ -4,20 +4,21 @@
 
 This release updates the Azure DevCenter library to use the 2022-11-11-preview API.
 
-### Features Added
+### Breaking Changes
 
 - `DevBoxClient`, `DevCenterClient`, and `EnvironmentsClient` now accept an endpoint URI on construction rather than tenant ID + dev center name.
```

Comment:
It actually the Builder takes an endpoint URI.

Constructor of these clients is not public.

Reply:
Fixed wording here.